app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/event_target_parser.rb

```diff
@@ -0,0 +1,34 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::EventTargetParser
+  attr_reader :ems_event
+
+  def initialize(ems_event)
+    @ems_event = ems_event
+  end
+
+  def parse
+    target_collection = InventoryRefresh::TargetCollection.new(
+      :manager => ems_event.ext_management_system,
+      :event   => ems_event
+    )
+
+    case ems_event[:event_type]
+    when /^pvm-instance\.create/
+      target_collection.add_target(
+        :association => :ext_management_system,
+        :manager_ref => {:ems_ref => ems_event.ext_management_system}
+      )
+    when /^pvm-instance\.update/
+      target_collection.add_target(
+        :association => :ext_management_system,
```

Comment:
Is there a way to target all existing VMs?

@vfebvre has a similar question [here](https://github.com/ManageIQ/manageiq-providers-ibm_power_hmc/pull/22/files#diff-fe5f4bef291f246bd6808eda3d3c8d1d8eceb696015b491ff70d1f7784c66a65R43).

Reply:
I think the way to go here would be to give a target that is just the `:association => :vms` with no `manager_ref`.

I don't know if this is possible at the `InventoryRefresh::Target` level, but assuming it is you could handle this case in your targeted_collector.  We do set the targeted_scope based on the manager_refresh_targets so you might have to be careful with that.

Reply:
For now I targeted the entire EMS (572010c), but in the future would be nice to only refresh VMs (or better yet, if the PowerVS event API will improve to allow us to target a single VM!)